### PR TITLE
Use ConfigMap for Maintenance Mode Flag

### PR DESCRIPTION
**Pull request by ivanELEC**
## Description of change

[Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1800)

- configuration.yaml adds a configmap for general settings we can change on the fly including MAINTENANCE_MODE
- MAINTENANCE_MODE configmap value use for environment variable 
- Maintenance mode feature uses environment variable
- enable_maintenance_mode/disable_maintenance_mode scripts can be ran locally if you have kubernetes auth, 1st arg is the namespace you want to run this in, 2nd arg is an optional BRANCH_RELEASE_NAME value which defaults to laa-assess-crime-forms-app

### CODE REVIEW

File: bin/disable_maintenance_mode [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+NAMESPACE=$1
+
+# Default to main deployment if 2nd arg not provided
+BRANCH_RELEASE_NAME=${2:-laa-assess-crime-forms-app}
+
+echo "Creating configmap with maintenance mode disabled"
+kubectl create configmap $BRANCH_RELEASE_NAME -n $NAMESPACE --from-literal=MAINTENANCE_MODE=false --dry-run=client -o yaml | kubectl -n $NAMESPACE apply -f -
+kubectl rollout restart deployment -n $NAMESPACE $BRANCH_RELEASE_NAME
```
Comment (jogilder) [inline bin/disable_maintenance_mode:10]: does this work in this order with namespace after deployment?

File: bin/enable_maintenance_mode [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+NAMESPACE=$1
+
+# Default to main deployment if 2nd arg not provided
+BRANCH_RELEASE_NAME=${2:-laa-assess-crime-forms-app}
+
+echo "Creating configmap with maintenance mode enabled"
+kubectl create configmap $BRANCH_RELEASE_NAME -n $NAMESPACE --from-literal=MAINTENANCE_MODE=true --dry-run=client -o yaml | kubectl -n $NAMESPACE apply -f -
+kubectl rollout restart deployment -n $NAMESPACE $BRANCH_RELEASE_NAME
```
[the remark this answers is not in the document]
Comment (ivanELEC) [reply bin/enable_maintenance_mode:10]: yes, I've tested this in a branch

**Review by jogilder — COMMENTED** [review]
see inline comments

**Review by jogilder — APPROVED** [review]
LGTM

**Pull request merged**